examples/src/agents/vectorstore_supabase.ts

```diff
@@ -0,0 +1,48 @@
+import { OpenAI } from "langchain";
+import { OpenAIEmbeddings } from "langchain/embeddings";
+import {
+  VectorStoreToolkit,
+  createVectorStoreAgent,
+  VectorStoreInfo,
+} from "langchain/agents";
+import { PGVectorStore } from "langchain/vectorstores";
+import { createClient } from '@supabase/supabase-js';
+
+
+export const run = async () => {
+  const model = new OpenAI({ temperature: 0 });
```

Comment:
**Question:** Might some users want a temp that is not 0? If so, should we allow them to pass in a custom temp as a part of the interface?